### PR TITLE
App Styling PR

### PR DESCRIPTION
**Pull request by AmaduKamara**
# Bookstore: styling

👨‍💻👨‍💻👨‍💻👨‍💻👨‍💻👨‍💻👨‍💻
Edited styles of the app to match exactly the one given in the sneak peek. 

To highlight:

✔️ Added the middle section that shows the progress status
✔️ Aligned the input fields
✔️ Added app screenshot in the README.md
✔️ Changed the font-size to be the body font-size from the design

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -2,3 +2,10 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+}
```
Comment (Mihndim2020) [inline src/index.css:11]: - [x] As it is a requirement for this project, and a required skill as a front end dev to stick to the  design, please kindly update your app to match with the design. Here are things to pay attention to; 
- The middle column in design is missing in your implementation(see screenshot below)
- The book `title input field`, `category field` and `add button` are not aligned(see screenshot below).

***The design***
theD

***Your implementation***
yourI

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @AmaduKamara, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Your project is running free of `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (AmaduKamara) [reply src/index.css:11]: Sure bro.
I have fixed things up as required and now the design looks just as required.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @AmaduKamara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [OPTIONAL] Kindly add the project screenshot, live link, and license to Readme file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaduKamara** [comment]
I actually added them but did that in the styling branch. 👍 Thanks, bro.

**Pull request merged**